jade/centralauth.py

```diff
@@ -9,10 +9,13 @@
 '''  # noqa: E501
 import mwapi
 
+from . import about
 from . import errors
 
+USER_AGENT = "{} -- {}".format(about.__name__, about.__author_email__)
 
-class CentralAuth:
+
+class CentralAuth(object):
```

Comment:
No longer necessary as of Python 3.0.  https://stackoverflow.com/questions/15374857/should-all-python-classes-extend-object